src/profile/encrypted/encryption/aes_gcm_256.rs

```diff
@@ -23,7 +23,7 @@ impl AesGcm256 {
 
 impl AesGcm256 {
     fn seal(
-        plaintext: Vec<u8>,
+        plaintext: impl AsRef<[u8]>,
```

Comment:
we should try to always use `impl AsRef<[u8]>` instead of `BagOfBytes` / `Vec<u8>` or `&[u8]` so that we can pass any :) can also pass `ExactlyNBytes` types etc.